pubspec.lock

```diff
@@ -7,7 +7,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.8.1"
+    version: "2.8.2"
```

Comment:
Why it is necessary to change package versions here?
Seems like you committed your local saved versions.